res/xml/device_maintainers_fragment.xml

```diff
@@ -1769,5 +1769,11 @@
             app:codename="@string/device_sunfish_codename"
             app:maintainer="@string/device_sunfish_maintainer"
             app:type="phone" />
+        <com.android.settings.rr.Preferences.DevicePreference
```

Comment:
Keep this under samsung not under google devices

Reply:
Done